dynamic-course-demo/workflow/src/main/java/org/acme/GreetingResource.java

```diff
@@ -0,0 +1,16 @@
+package org.acme;
```

Comment:
pls delete the custom code and retain only the minimal workflow-related resources